www/js/views/submit.js

```diff
@@ -86,7 +86,7 @@
                 if ( !this._handleInvalid( model, err, options ) ) {
                     var errors = err.errors;
                     var errorList = '<ul><li class="plain">' + FMS.strings.invalid_report + '</li>';
-                    var validErrors = [ 'password', 'category', 'name' ];
+                    var validErrors = [ 'password', 'password_register', 'category', 'name' ];
```

Comment:
Sorry, wonder why I didn't use password – I guess it appears in a different part of the web interface.